### PR TITLE
Pull Request for code reviewing of the 5th project

### PR DESCRIPTION
**Pull request by abouhid**
This pull request aims to submit our project to be reviewed, approved, and later merged to the master branch.

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hi @abouhid and @Rocio01 👋. Excellent project team 👏👌! You have done a remarkable job making this project. Killer README btw 😎.

### But in order to get it approved we need to work on changes. Let's see,

- There is a horizontal scrollbar on your page. Let's make sure to remove that.

Annotation 2020-06-28 211252

`article.click` is one of elements that are causing the above issue. You can this [useful tool](https://chrome.google.com/webstore/detail/pesticide-for-chrome/bblbgcheenepgnnajgfpiicnbbdmmooh) to debug these kinds of issues.

#### *Remember to NOT use* `overflow-x: hidden;` *as a solution.*

- You need to focus on the **heatmap** of the original site and not on making a grayscale clone of it. It means that you just need to highlight where and how each element sits on the page. You don't need to style them. Think of it this way: you are a house planner and you need to tell the builder where in the house a bathroom/bedroom/kitchen should be and what are their dimensions. You can use borders to indicate each element. In the case of images, use placeholders i.e., `divs` with height, width and background color. 

- Add borders here.

Annotation 2020-06-28 212552

- The footer has two images on either side on the original site. Please make sure to add their placeholders in your heatmap.

#### Heatmap
Annotation 2020-06-28 212632

#### Original
Annotation 2020-06-28 212702

- There is a warning when your HTML is checked in the validator. 

image

You can add the element and set it's `display: none`. Remember about NOT using in-line styles.

- I have also noticed some invalid CSS properties. Check the comments below.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+  text-decoration: none;
+  cursor: pointer;
+}
+
+h5,
+a,
+p,
+h2,
+img {
+  padding: 20px auto;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: `padding` doesn't accept `auto` as one of it's values.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,799 @@
...
+}
+
+.see {
+  margin: 60px auto;
+  color: black;
+  border-bottom: 10px solid white;
+}
+
+.click {
+  margin-top: 8% auto;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: `margin-top` only accepts one value.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply these changes and request another code review.

Happy coding 👨‍💻

**Comment by abouhid** [comment]
Issues addressed! Thank you for your quick and assertive response!

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hi, it's me again 😃. Thank you for making the changes.

### However,

- There is still a horizontal scrollbar.

image

`footer-imgs` is the cause now.

- Remember to use **placeholders** in place of images, not the actual image.

- You have missed one issue.

Annotation 2020-06-28 224127


Apply these changes and request another code review.

Happy coding 👨‍💻

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Almost there team. There is one small change that you missed.

#### [REQUIRED]
Previously I had requested you to put placeholders instead of actual images. Please make sure of that. Adding images to your heatmap defeats the purpose because the images might change and the front-end dev who will be designing the site might get confused. Heatmaps are guides for front-end devs for designing the actual site. That's why this will be good for you to understand why it's important.

#### [OPTIONAL]

Consider fixing this:
Annotation 2020-06-28 233200

The original:
Annotation 2020-06-28 233225

Apply this small change and request another code review.

Happy coding 👨‍💻

**Comment by abouhid** [comment]
I am sorry, it took me a while to understand what you meant by placeholders! Now I think it is correct!

**Review by elmejdki — APPROVED** [review]
# status: Approve :star: :star: :star: :star: :star:
## Great job guys :clap: , you have made a really good clone of the original smashing magazine website :tophat: .

now you can merge to master your project approved.

## Happy Coding :tophat:.

**Pull request merged**